floor/floor/processor/kaleidoscope.py

```diff
@@ -46,7 +49,7 @@ def get_next_frame(self, context):
             for y in range(0, 8):
                 index = y*8 + x
                 if self.times[index] > 0:
-                    age_scale = 1 - (t - self.times[index])/self.LIFETIME
+                    age_scale = 1 - old_div((t - self.times[index]),self.LIFETIME)
```

Comment:
Ok, maybe all these old_div should be '/' since I don't think anyone planned on having '/' do a floor div.